src/client/app/components/visual-unit/VisualUnitDetailComponent.tsx

```diff
@@ -0,0 +1,68 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { FormattedMessage } from 'react-intl';
+import TooltipHelpComponent from '../TooltipHelpComponent';
+import * as React from 'react';
+import { CreateVisualUnitComponent } from './CreateVisualUnitComponent';
+import TooltipMarkerComponent from '../TooltipMarkerComponent';
+import { selectCik } from '../../redux/api/conversionsApi';
+import { selectConversionsDetails } from '../../redux/api/conversionsApi';
+import { useAppSelector } from '../../redux/reduxHooks';
+import { selectAllUnits } from '../../redux/api/unitsApi';
+
+/**
+ * Defines the units and conversion graphics view.
+ * @returns Units visual graphics page element
+ */
+export default function VisualUnitDetailComponent() {
+	/* Get unit and conversion data from redux */
+	const unitData = useAppSelector(selectAllUnits);
```

Comment:
unitData only seems used to pass to the units to the visual components. Since those are components, would it make sense to move this there and not pass since both types of visuals use the same units?